test/parallel/test-util.js

```diff
@@ -153,3 +154,21 @@ util.print('test');
 util.puts('test');
 util.debug('test');
 util.error('test');
+
+{
+  // binding.isNativeError()
+  assert.strictEqual(binding.isNativeError(new Error()), true);
+  assert.strictEqual(binding.isNativeError(new TypeError()), true);
+  assert.strictEqual(binding.isNativeError(new SyntaxError()), true);
+  /* eslint-disable new-parens */
+  assert.strictEqual(binding.isNativeError(new (context('Error'))), true);
```

Comment:
`new (context('Error'))()`

I think that should fulfill the "new-parens" ESLint rule as well.